packages/react-portal/src/virtualParent/setVirtualParent.ts

```diff
@@ -0,0 +1,22 @@
+import { VirtualElement } from './Types';
```

Comment:
```suggestion
import { VirtualElement } from './types';
```

_nit_: can we normalize casing there?